### PR TITLE
[#367] Once logged in the form allow user go back to the homepage

### PR DESCRIPTION
**Pull request by tangrammer**
and a couple of minor things/commits included

### CODE REVIEW

File: frontend/src/modules/signup-old/view.jsx [hunk 1]
```diff
@@ -68,8 +82,12 @@ const SignupView = ({ ...props }) => {
                     className="green"
                     onClick={() => {
                       setTypeSignUp("stakeholder");
-                      loginWithPopup({ screen_hint: "signup" });
-                      //                  history.push("/entity-signup");
+                      console.log(profile.about, Boolean(profile.about));
+                      if (isRegistered(profile)) {
```
Comment (wayangalihpratama) [inline frontend/src/modules/signup-old/view.jsx:86]: Need to make sure of this condition,
I think this if condition should be `if (!isRegistered(profile))`, then go to register?

Comment (tangrammer) [reply frontend/src/modules/signup-old/view.jsx]: thanks for catching that!

Comment (tangrammer) [reply frontend/src/modules/signup-old/view.jsx]: finally using isAuthenticated

**Review by wayangalihpratama — APPROVED** [review]
Looks great!

**Pull request merged**